pkg/apis/networking/v1/types.go

```diff
@@ -19,16 +19,18 @@ package v1
 type NetworkType string
 
 const (
-	NetworkTypeUnderlay = NetworkType("Underlay")
-	NetworkTypeOverlay  = NetworkType("Overlay")
+	NetworkTypeUnderlay  = NetworkType("Underlay")
+	NetworkTypeOverlay   = NetworkType("Overlay")
+	NetworkTypeGlobalBGP = NetworkType("GlobalBGP")
```

Comment:
no need for a new network type, just need one new mode

Reply:
Actually we do need one, because we don't have the capability to specify network node while creating pods.

Reply:
I found that networkModeGlobalBGP is used in daemon part, and networkTypeGlobalBGP is used in webhook, how about treating globalBGP as one kind of underlay type? And use a function named `IsGlobalBGPMode` instead of introducing a new network type?